source/constants.js

```diff
@@ -1,5 +1,6 @@
 exports.SAFE_DELIMITER = "→";
 exports.MAX_SET_TIMEOUT = 2 ** 31 - 1;
+exports.YEAR_IN_MS = 31556926000;
```

Comment:
Are we sure that there isn't something within JS by default with this value, so we don't have to have a random constant for this?

Reply:
Happy to use it if you can find it, I couldn't.